src/lib/features/project-environments/environment-service.ts

```diff
@@ -254,22 +269,13 @@ export default class EnvironmentService {
         const projectEnvs =
             await this.projectStore.getEnvironmentsForProject(projectId);
 
-        if (projectEnvs.length > 1) {
-            await this.forceRemoveEnvironmentFromProject(
+        await this.forceRemoveEnvironmentFromProject(environment, projectId);
+        await this.eventService.storeEvent(
+            new ProjectEnvironmentRemoved({
+                project: projectId,
                 environment,
-                projectId,
-            );
-            await this.eventService.storeEvent(
-                new ProjectEnvironmentRemoved({
-                    project: projectId,
-                    environment,
-                    auditUser,
-                }),
-            );
-            return;
-        }
-        throw new MinimumOneEnvironmentError(
```

Comment:
Removed this validation, we should no longer need this

Reply:
Would be nice to remove the validation class, can be done as a follow up if you prefer, but this is probably the last place we used it

Reply:
Cheers, good call, removed